test/integration/capybara/observations_controller_supplemental_test.rb

```diff
@@ -90,6 +90,17 @@ def test_observation_edit_uncheck_species_list
     assert_not_includes(species_list.observations, observation)
   end
 
+  def test_observation_remove_collection_number
```

Comment:
In which file does this test belong?

Reply:
This one is fine. 
For MO's integration tests, they seem to have started out with one of two organizing principles: either by use case or by template. This fits both.